### PR TITLE
Implemented Menu and Shop Use Cases

### PR DESCRIPTION
**Pull request by stephlum**

### CODE REVIEW

File: src/businessrules/dai/ShopRepository.java [hunk 1]
```diff
@@ -3,5 +3,15 @@
...
 public interface ShopRepository {
+    String createShop(JSONObject data);
+
     boolean updateShop(String shopId, JSONObject shop);
+
+    boolean clearShopMenu(String shopId);
+
+    JSONObject readShop(String shopId);
+
+    JSONObject readShopMenu(String shopId);
```
Comment (VijayS02) [inline src/businessrules/dai/ShopRepository.java:14]: Why is a read shop menu required, can you not just read shop then pull the menu from that?

Comment (akashilangovan) [reply src/businessrules/dai/ShopRepository.java:14]: Agreed

**Review by akashilangovan — APPROVED** [review]

File: src/businessrules/dai/ShopRepository.java (hunk 1 above)
Comment (stephlum) [reply src/businessrules/dai/ShopRepository.java:14]: Do you mean whenever we need to read the Shop Menu we would just call readShop and then parse to get the shop json object within that method? would it be like:
`JSONObject shopData = shopRepository.readShop.getJSONobject("shop")`

**Review by akashilangovan — APPROVED** [review]
Lgtm

**Pull request merged**